package.json

```
@@ -15,8 +15,8 @@
     "lint:js": "eslint --ext '.js,.ts,.tsx' src/ example/src FabricExample/src MacOSExample/src && yarn prettier --check './{src,example,FabricExample,MacOSExample}/**/*.{js,jsx,ts,tsx}'",
     "lint:js-root": "eslint --ext '.js,.ts,.tsx' src/ && yarn prettier --check './src/**/*.{js,jsx,ts,tsx}'",
     "lint:android": "./android/gradlew -p android spotlessCheck -q",
-    "checkIntegrity": "(cd ./FabricExample/android && ./gradlew generateCodegenArtifactsFromSchema -PskipCodegenCopyTask) && (cd ./android && ./gradlew checkIntegrityBetweenArchitectures)",
-    "circularDependencyCheck": "yarn madge --extensions js,ts,tsx --circular src"
+    "check-integrity": "(cd ./FabricExample/android && ./gradlew generateCodegenArtifactsFromSchema -PskipCodegenCopyTask) && (cd ./android && ./gradlew checkIntegrityBetweenArchitectures)",
```

```suggestion
    "integrity-check": "(cd ./FabricExample/android && ./gradlew generateCodegenArtifactsFromSchema -PskipCodegenCopyTask) && (cd ./android && ./gradlew checkIntegrityBetweenArchitectures)",
```

We do have `ts-check` and `circular-dependency-check`, so I think we could name it `integrity-check` (cc @j-piasecki)

---

i agree

---

I'd go even further and change it to `cross-arch-integrity-check` - it must have slipped past the original review, as the name `checkIntegrity ` doesn't really say much. (Integrity of what?)

---

Sure, especially since `integrity-check` is used by yarn already: `yarn install ... --skip-integrity-check`